bokehjs/src/lib/models/axes/axis.ts

```diff
@@ -380,6 +387,7 @@ export namespace Axis {
     major_tick_out: number
     minor_tick_in: number
     minor_tick_out: number
+    fixed_location: any
```

Comment:
It would have been useful to have this filled in at this point (`float | null`).

Reply:
good point, the `p.Any` threw me off (tho `float | string | null`) I think

Reply:
So, based on another comment, if `string` is for categoricals, then this should be `Factor` (from `models/ranges/factor_range`) instead.

Reply:
So, `number | Factor | null`.

Reply:
I'm not sure where I took `float` from.

Reply:
unfortunately so far  trying to change this goes down a rabbit hole of type problems.